demo/client/src/ml-models/deploy-model-node.js

```diff
@@ -91,7 +93,9 @@ async function deploySparsePerceptron(model, web3, toFloat) {
     }
 }
 
-async function deployNearestCentroidClassifier(model, web3, toFloat) {
+async function deployNearestCentroidClassifier(model, web3, options){
+    const {toFloat, initialChunkSize = 200, chunkSize = 250,
```

Comment:
Chunk sizes aren't used. They don't need to be now but we shouldn't initialize them if they're not used.